[2021-09-27 15:51:36,460] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: day1_challenge2.print_log_message 2021-09-27T15:40:00+00:00 [queued]>
[2021-09-27 15:51:36,481] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: day1_challenge2.print_log_message 2021-09-27T15:40:00+00:00 [queued]>
[2021-09-27 15:51:36,482] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 15:51:36,484] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-09-27 15:51:36,486] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 15:51:36,516] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): print_log_message> on 2021-09-27T15:40:00+00:00
[2021-09-27 15:51:36,540] {standard_task_runner.py:52} INFO - Started process 35714 to run task
[2021-09-27 15:51:36,562] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'day1_challenge2', 'print_log_message', '2021-09-27T15:40:00+00:00', '--job-id', '103', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/day1_challenge2.py', '--cfg-path', '/tmp/tmp6ttpnym7', '--error-file', '/tmp/tmpt8o_l74m']
[2021-09-27 15:51:36,584] {standard_task_runner.py:77} INFO - Job 103: Subtask print_log_message
[2021-09-27 15:51:36,791] {logging_mixin.py:109} INFO - Running <TaskInstance: day1_challenge2.print_log_message 2021-09-27T15:40:00+00:00 [running]> on host 196580edc338
[2021-09-27 15:51:37,150] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Attasil Chitwirote
AIRFLOW_CTX_DAG_ID=day1_challenge2
AIRFLOW_CTX_TASK_ID=print_log_message
AIRFLOW_CTX_EXECUTION_DATE=2021-09-27T15:40:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-27T15:40:00+00:00
[2021-09-27 15:51:37,161] {python.py:151} INFO - Done. Returned value was: None
[2021-09-27 15:51:37,259] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=day1_challenge2, task_id=print_log_message, execution_date=20210927T154000, start_date=20210927T155136, end_date=20210927T155137
[2021-09-27 15:51:37,465] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-27 15:51:38,093] {local_task_job.py:261} INFO - 1 downstream tasks scheduled from follow-on schedule check
